index.test.js

```diff
@@ -53,6 +56,18 @@ describe('handler', () => {
 
     const event4 = event('/use1-whatev/1234/adfree/some-guid/some-digest/file.mp3');
     expect(handler(event4)).toEqual(event4.request);
+
+    const event5 = event('/t/some.domain.com/test.wav/72.1.16000.m4a');
```

Comment:
Hrmm.. should these be a different test?  Seems like it's not the "short/fake looking digests" bit of the code you're hitting here.

Reply:
those are tests for non-expiring links, not short/fake urls?